Makefile

```diff
@@ -1,78 +1,51 @@
-#!/usr/bin/env make
+MOUNT_TARGET_DIRECTORY  = /app/src
+BUILD_TOOLS_DOCKER_REPO = mineiros/build-tools
+DOCKER                  = $(shell which docker)
```

Comment:
this is a noop....? so you can use `docker` right away it willl fail no matter what?

Reply:
ok, I've changed that